docs/BindingsGuide_DotnetOutOfProc.md

```diff
@@ -0,0 +1,187 @@
+# SQL Extension .NET Worker
+
+Welcome to the Sql Extension .NET Worker Repository. The .NET Worker provides .NET 6 support for SQL Bindings in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline. A .NET Isolated function app works differently than a .NET Core 3.1 function app. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-outofproc/Program.cs) that starts the worker.
+
+## Binding Model
+
+.NET Isolated introduces a new binding model, slightly different from the binding model exposed in .NET Core 3 Azure Functions. More information can be [found here](https://github.com/Azure/azure-functions-dotnet-worker/wiki/.NET-Worker-bindings). Please review our samples for usage information.
+
+## Samples
+
+You can find samples on how to use Sql Extension .NET Worker under `samples\samples-outofproc` ([link](https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-outofproc)).
+
+## Create and run .NET Isolated functions
+
+**Note: Visual Studio and Visual Studio Code support is on the way. In the meantime, please use `azure-functions-core-tools` or the sample projects as a starting point.**  
+
+### Install .NET 6.0
+
+Download .NET 6.0 [from here](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+### Install the Azure Functions Core Tools
+
+To download Core Tools, please check out our docs at [Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools)
+
+### Create an Isolated Function App
+
+Run the below in the Terminal/CLI or with PowerShell.
+
+    ```
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime dotnet (Isolated Process)
+    ```
+
+   or in an empty directory, run `func init` and select `dotnet (Isolated Process)`
+
+### Add a function: 
```

Comment:
Remove this - adding functions isn't special to out of proc so we can just link to the normal .NET guide at the end of this